mozilla/foundation.mozilla.org
# full-width image blocks on blog page

Closes #6676 

**Link to sample test page**:[ Link to Test Blog Page](https://foundation-s-full-width-ct581i.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/)

Steps to test:
1.) Click link above to visit a test blog page
2.) Note the full-width image, and how it responds to different screen sizes


**Note**: The wagtail tests are going to fail as I have not added the migrations to source control yet, once approved I will pull the latest migrations from master, run makemigrations, and then merge them in to ensure that they are up to date


## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/master/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?
